tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestExampleGenerator.java

```diff
@@ -768,6 +768,25 @@ public void TestDateFormatsWithContext() {
         assertEquals("Currency format example faulty", "【٥‏/٩‏/١٩٩٩〗【⃪٥‏/٩‏/١٩٩٩〗", actual);
     }
 
+    public void TestDateTimeComboFormats() {
+        ExampleGenerator exampleGenerator = getExampleGenerator("en");
+        checkValue(
+                "DateTimeCombo long std",
+                "〖❬September 5, 1999❭, ❬1:25:59 PM Eastern Standard Time❭〗〖❬September 5, 1999❭, ❬1:25 PM❭〗〖❬September 5, 1999❭, ❬7:00 AM – 1:25 PM❭〗〖❬today❭, ❬7:00 AM – 1:25 PM❭〗",
+                exampleGenerator,
+                "//ldml/dates/calendars/calendar[@type=\"gregorian\"]/dateTimeFormats/dateTimeFormatLength[@type=\"long\"]/dateTimeFormat[@type=\"standard\"]/pattern[@type=\"standard\"]");
+        checkValue(
+                "DateTimeCombo short std",
+                "〖❬9/5/99❭, ❬1:25:59 PM Eastern Standard Time❭〗〖❬9/5/99❭, ❬1:25 PM❭〗〖❬9/5/99❭, ❬7:00 AM – 1:25 PM❭〗〖❬today❭, ❬7:00 AM – 1:25 PM❭〗",
```

Comment:
It seems like people would more likely use the abbreviated timezone with the numeric date. Would you want both?
❬9/5/99❭, ❬1:25:59 PM Eastern Standard Time❭〗
This isn't a blocker, so approving.

Reply:
I thought there would potentially be more grammatical issues with the full timezone, so that is why I used that.